test/utils/verifyStateShape.spec.js

```diff
@@ -0,0 +1,13 @@
+import expect from 'expect';
+import verifyStateShape from '../../src/utils/verifyStateShape';
+
+describe('verifyStateShape', () => {
+  it('should return warning message if object shapes differ', () => {
```

Comment:
I'd rather see the unit test of `combineReducers` having correct behavior, to make sure it's actually being used correctly.